### PR TITLE
Add new snippets for LINQ SetOperation

### LINKED ISSUE #15951: Examples?
Probably put some use examples in as well to keep this page consistent with the rest of the documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fbdace9d-6fa0-0aec-25b4-df6f414001d7
* Version Independent ID: 77eff122-b9a5-2b1f-3a68-df302cb46194
* Content: [Set Operations (C#)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/set-operations#feedback)
* Content Source: [docs/csharp/programming-guide/concepts/linq/set-operations.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/concepts/linq/set-operations.md)
* Product: **dotnet-csharp**
* Technology: **csharp-linq**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by SuperNova911**
Link snippets from dotnet/sample [this](https://github.com/dotnet/samples/pull/1804#issue-346512804) pull request

Fixes #15951

### CODE REVIEW

File: docs/csharp/programming-guide/concepts/linq/set-operations.md [hunk 1]
```diff
@@ -20,24 +20,33 @@ Set operations in LINQ refer to query operations that produce a result set that
 ## Comparison of Set Operations  
   
 ### Distinct  
- The following illustration depicts the behavior of the <xref:System.Linq.Enumerable.Distinct%2A?displayProperty=nameWithType> method on a sequence of characters. The returned sequence contains the unique elements from the input sequence.  
+ The following example depicts the behavior of the <xref:System.Linq.Enumerable.Distinct%2A?displayProperty=nameWithType> method on a sequence of characters. The returned sequence contains the unique elements from the input sequence.  
   
  ![Graphic showing the behavior of Distinct&#40;&#41;.](./media/set-operations/distinct-method-behavior.png)  
+ 
+ [!code-csharp[Distinct](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#1)]
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/concepts/linq/set-operations.md]:
```suggestion
 [!code-csharp-interactive[Distinct](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#1)]
```

File: docs/csharp/programming-guide/concepts/linq/set-operations.md [hunk 2]
```diff
@@ -20,24 +20,33 @@ Set operations in LINQ refer to query operations that produce a result set that
...
   
 ### Except  
- The following illustration depicts the behavior of <xref:System.Linq.Enumerable.Except%2A?displayProperty=nameWithType>. The returned sequence contains only the elements from the first input sequence that are not in the second input sequence.  
+ The following example depicts the behavior of <xref:System.Linq.Enumerable.Except%2A?displayProperty=nameWithType>. The returned sequence contains only the elements from the first input sequence that are not in the second input sequence.  
   
  ![Graphic showing the action of Except&#40;&#41;.](./media/set-operations/except-behavior-graphic.png "Shows the behavior of Except.")  
   
+[!code-csharp[Except](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#2)]
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/concepts/linq/set-operations.md]:
```suggestion
[!code-csharp-interactive[Except](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#2)]
```

File: docs/csharp/programming-guide/concepts/linq/set-operations.md [hunk 3]
```diff
@@ -20,24 +20,33 @@ Set operations in LINQ refer to query operations that produce a result set that
...
+
 ### Intersect  
- The following illustration depicts the behavior of <xref:System.Linq.Enumerable.Intersect%2A?displayProperty=nameWithType>. The returned sequence contains the elements that are common to both of the input sequences.  
+ The following example depicts the behavior of <xref:System.Linq.Enumerable.Intersect%2A?displayProperty=nameWithType>. The returned sequence contains the elements that are common to both of the input sequences.  
   
  ![Graphic showing the intersection of two sequences.](./media/set-operations/intersection-two-sequences.png)  
  
+[!code-csharp[Intersect](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#3)]
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/concepts/linq/set-operations.md]:
```suggestion
[!code-csharp-interactive[Intersect](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#3)]
```

File: docs/csharp/programming-guide/concepts/linq/set-operations.md [hunk 4]
```diff
@@ -20,24 +20,33 @@ Set operations in LINQ refer to query operations that produce a result set that
...
+
 ### Union  
- The following illustration depicts a union operation on two sequences of characters. The returned sequence contains the unique elements from both input sequences.  
+ The following example depicts a union operation on two sequences of characters. The returned sequence contains the unique elements from both input sequences.  
   
  ![Graphic showing the union of two sequences.](./media/set-operations/union-operation-two-sequences.png)  
+
+[!code-csharp[Union](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#4)]
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/concepts/linq/set-operations.md]:
```suggestion
[!code-csharp-interactive[Union](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQSetOperation/CS/SetOperation.cs#4)]
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
As @BillWagner stated in https://github.com/dotnet/docs/issues/15951#issuecomment-557961289, these samples seem to be good to have the interactive experience.

**Comment by SuperNova911** [comment]
Change `code-csharp` tag to `code-csharp-interactive`

**Review by BillWagner — APPROVED** [review]
I like the way you've incorporated the new snippets into the existing explanations.

I'll :shipit: now.

**Pull request merged**